doc/vim-jsonnet.txt

```diff
@@ -0,0 +1,105 @@
+     #    ### #     #        #####   #####  ####### #     # #     # #######    #
+====================================================================================
+CONTENTS                                                         *jsonnet-contents*
+
+  1. Intro........................................|jsonnet-intro|
+  2. Install......................................|jsonnet-install|
+  3. Commands.....................................|jsonnet-commands|
+  4. Mappings.....................................|jsonnet-mappings|
+  6. Functions....................................|jsonnet-functions|
+  7. Settings.....................................|jsonnet-settings|
+  8. Troubleshooting..............................|jsonnet-troubleshooting|
+  9. Credits......................................|jsonnet-credits|
+
+==============================================================================
+INTRO                                                               *jsonnet-intro*
+
+==============================================================================
+INSTALL                                                           *jsonnet-install*
+
+==============================================================================
+COMMANDS                                                         *jsonnet-commands*
+
+
+                                                                      *:JsonnetFmt*
+:JsonnetFmt
+
+Filter the current Jsonnet buffer through `jsonnet fmt`.  It tries to
+preserve cursor position and avoids replacing the buffer with stderr
+output.
+
+==============================================================================
+MAPPINGS                                                        *jsonnet-mappings*
+
+==============================================================================
+FUNCTIONS                                                       *jsonnet-functions*
+
+                                                                 *jsonnet#Format()*
+
+Filter the current Jsonnet buffer through `jsonnet fmt`.  It tries to
+preserve cursor position and avoids replacing the buffer with stderr
+output.
+
+
+==============================================================================
+SETTINGS                                                        *jsonnet-settings*
+
+                                                         *'g:jsonnet_fmt_on_save'*
+
+Use this option to auto |:JsonnetFmt| on save. By default it's enabled >
+
+  let g:jsonnet_fmt_on_save = 1
+<
+                                                             *'g:jsonnet_command'*
+
+Use this option to define which tool is used to jsonet fmt. By default `jsonnet` is
+used >
+
+  let g:jsonnet_command = "jsonnet"
+<
+                                                         *'g:jsonnet_fmt_command'*
+
+Use this option to define which <cmd> parameter is used to *g:jsonnet_command* tool.
+By default `fmt` isused >
```

Comment:
Thanks, I have addressed these comments. 
Special thanks for the diligent review paying attention to typos.